scripts/xcpng-fs-diff.py

```diff
@@ -250,15 +250,18 @@ def main():
     folders = ["/boot", "/etc", "/opt", "/usr"]
     ignored_file_patterns = [
         '/boot/initrd-*',
+        '/boot/efi/*',
```

Comment:
I assume this is so that comparing a UEFI host with a BIOS can be done?

Reply:
Right. I need to check if there is no other difference between UEFI and BIOS hosts but this is the only one that I remember.